src/EA.Weee.Web/Areas/Aatf/Mappings/ToViewModel/ManageEvidenceNoteViewModelMap.cs

```diff
@@ -39,6 +42,9 @@ public ManageEvidenceNoteViewModel Map(ManageEvidenceNoteTransfer source)
                 model.SubmittedDatesFilterViewModel = source.SubmittedDatesFilterViewModel;
             }
 
+            model.ComplianceYearList = source.ComplianceYearList;
```

Comment:
@GuidoCangelosi  This needs unit tests, add tests to ManageEvidenceNoteViewModelMapTests to check the complianceYearList property and SelectedComplianceYears are set as expected